INN/umbrella-currentorg
# Allow selecting empty parent category for tags

## Changes

This pull request makes the following changes:

- `hide_empty` was set to true in the `wp_dropdown_categories` args for the tag parent category dropdown, not allowing you to select an empty category as a parent for a tag.

## Why



For #23.

## Testing/Questions

Features that this PR affects:

- Category specific tags.


Questions that need to be answered before merging:

- [ ] Can you now see empty categories in the dropdowns?

Steps to test this PR:

1. Go to a WPBDP tag, open the parent category dropdown, and verify categories with no posts are in the dropdown.
